completions/lzip

```diff
@@ -14,14 +14,14 @@ _lzip()
             --volume-size | --data-size | -${noargopts}@([bmsSB]|[hV]*))
             return
             ;;
-        --decompress-${noargopts}d)
+        --decompress-d)
```

Comment:
This reads weird to me, but then again so did the original. I suppose it should be:
```suggestion
        --decompress | -${noargopts}d)
```

Similarly for `--threads` and `--output` below.

Reply:
Ah, OK. This was a revert commit for https://github.com/scop/bash-completion/pull/862/files#diff-87387e55a9536f7ac2aa5fce39d9a0442c29ec2058215563388d5a284a9ac1c1, but now I realized what was the problem after checking the man page of `lzip`: `|` was actually missing in the original codes.

I modified the commit. You can also cherry-pick this commit 7467419.

Reply:
Thanks! Cherry-picked.